geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/export/ExportParams.scala

```diff
@@ -51,9 +55,18 @@ trait BinExportParams extends BaseExportParams {
   @Parameter(names = Array("--lon-attribute"), description = "Name of the longitude attribute to export")
   var lonAttribute: String = null
 
-  @Parameter(names = Array("--dt-attribute"), description = "Name of the date attribute to export", required = true)
-  var dateAttribute: String = null
-
   @Parameter(names = Array("--label-attribute"), description = "Name of the attribute to use as a bin file label")
   var labelAttribute: String = null
 }
+
+trait OptionalBinExportParams extends BaseBinExportParams {
+  @Parameter(names = Array("--dt-attribute"), description = "Name of the date attribute to export")
+  var dateAttribute: String = null
+}
+
+trait RequiredBinExportParams extends BaseBinExportParams {
+  @Parameter(names = Array("--dt-attribute"), description = "Name of the date attribute to export", required = true)
+  var dateAttribute: String = null
```

Comment:
i think date attribute shouldn't be required for bin exports - it should default to the simple feature type default date